pkg/cluster/spec/validate.go

```diff
@@ -460,6 +474,61 @@ func (s *Specification) platformConflictsDetect() error {
 	return nil
 }
 
+func (s *Specification) portInvalidDetect() error {
+	portTypes := []string{
```

Comment:
How about just iterate every reflected field and apply check process if it's name like `XXXPort`

Reply:
Thanks for the advice